### ISSUE TITLE
download_dependencies.sh throws error on osx 10.10 - syntax issue

### ISSUE THREAD

**Issue by unsalted** [tensorflow/tensorflow#4710]
I receive this error on osx 10.10

```
$ sh tensorflow/contrib/makefile/download_dependencies.sh
tensorflow/contrib/makefile/download_dependencies.sh: line 51: syntax error near unexpected token `<'
```
It was fixed by changing line 51 to: 
```
curl -Ls "${url}" | tar -C "${dir}" --strip-components=1 -xz
```

**Comment by unsalted** [comment]
PR #4711

**Comment by unsalted** [comment]
Fixed with PR #4711
